src/libraries/System.Runtime/tests/System.Globalization.Tests/System/Globalization/RegionInfoTests.cs

```diff
@@ -266,5 +267,19 @@ public void ValidateThrowingWhenUsingCustomUnspecifiedLcid()
 
             AssertExtensions.Throws<ArgumentException>("culture", () => new RegionInfo(4096));
         }
+
+        [Fact]
+        public void BuiltInRegionListTest()
+        {
+            // Ensure we can create all region info objects from the built-in list
+            Dictionary<string, string> regionNames = (Dictionary<string, string>)typeof(RegionInfo).Assembly.GetType("System.Globalization.CultureData").GetProperty("RegionNames", BindingFlags.Static | BindingFlags.NonPublic).GetValue(null);
```

Comment:
Is there another source we could be enumerating? e.g. how does this related to the cultures enumerable from CultureInfo.GetCultures(All)... if we were to parse the regions out of those, would that be equivalent to (some of) this data?

Reply:
Culture enumeration can be done too but in our data, we add some data that is not getting from the enumeration. That is why I decided to do use the data we build.